umap/locale/cs_CZ/LC_MESSAGES/django.po

```diff
@@ -227,7 +228,7 @@ msgstr "Vyberte poskytovatele mapy"
 msgid ""
 "uMap lets you create maps with <a href=\"%(osm_url)s\" />OpenStreetMap</a> "
 "layers in a minute and embed them in your site."
-msgstr ""
+msgstr "uMap umožňuje vytvářet mapy s vrstvami <a href=\"%(osm_url)s\" />OpenStreetMap</a> za okamžik a vložit je do svých stránek."
```

Comment:
The translator might wish to level up a bit and create sentences that make sense.

Reply:
I encourage you to "hack" translation  in transifex 👍 
https://www.transifex.com/yohanboniface/umap/dashboard/

Reply:
I don't speak CZ, so I can't help you 😼

Reply:
As advised by @yohanboniface, i will made a "backup" of CZ transifex translation. Then, i will try to push translations from the code to transifex.

Next, i will check for each language.

Reply:
Thank you! Please tag me if you want me to re-check once you update things :)

Reply:
Done :+1: